### ISSUE TITLE
Import modal: could not upload ".jpynb" file type when accept set to "text/*" or "text/plain"

### ISSUE THREAD

**Issue by jillyj** [carbon-design-system/ibm-cloud-cognitive#1933]
## What package(s) are you using?

- [x] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [ ] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.

Unable to upload the file with extension `.jpynb` from file explorer or drag and drop. The accept prop is set to `accept: ["text/*", "text/plain"]`. The error `Invalid file type` occurs.

> Is this issue related to a specific component?

Yes, Import Modal

> What did you expect to happen? What happened instead? What would you like to
> see changed?

- Should be able to upload the file with plain text no matter the file extension is txt or not with `accept: ["text/plain"]`.
- Should be able to upload the file no matter the file extension with `accept: ["text/*"]`.
- Should be able to upload any file type with  `accept: ["*/*"]`.

> What browser are you working in?

Chrome, Firefox

> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?

@carbon/ibm-products 1.15.0

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

CP4S Data Explorer

## Steps to reproduce the issue
1. Unzip [test.ipynb.zip](https://github.com/carbon-design-system/ibm-cloud-cognitive/files/8521145/test.ipynb.zip)
2. Drag and drop this file or upload this `test.jpynb` file form the sandbox example below.
 => The error `Invalid file type` occurs.

> Please create a reduced test case in CodeSandbox
>
[> https://codesandbox.io/s/github/carbon-design-system/ibm-cloud-cognitive/tree/main/examples/cloud-cognitive/codesandbox](https://codesandbox.io/s/naughty-star-1x1gn1?file=/src/Example/Example.js)

## Additional information

- Screenshots or code

**Comment by jillyj** [comment]
Is it possible to fix this issue soon? We are going to use this component soon. Many thanks.

**Comment by matthewgallo** [comment]
@davidmenendez would you happen to know what's going on with this?

**Comment by davidmenendez** [comment]
hey @jillyj apologies for the late reply. so the underlying component is just the carbon components file uploader (https://react.carbondesignsystem.com/?path=/docs/components-fileuploader--file-uploader). if you're trying to just allow any file type it looks like you can just set accept to an empty array `accept: []`. see if that works for you.

**Comment by jillyj** [comment]
Yes, set an empty array to accept works~ Not sure why it doesn't work while I tried this way before. Thanks!
